### ISSUE TITLE
How do I combine matched features with Colmap

### ISSUE THREAD

**Issue by geovsion** [ubc-vision/COTR#10]

**Comment by geovsion** [comment]
hello, i want to know that how i can put your correspondences into colmap, thanks

**Comment by jiangwei221** [comment]
COLMAP requires steady keypoints across images, I'd use the code example in guided matching to obtain matches with consistent keypoints.

**Comment by geovsion** [comment]
Thanks! But I still have a problem, how to speed up the matching process, it is too slow, despite tuning the relevant parameters.

**Comment by jiangwei221** [comment]
Yes, currently we are also working on the speed problem, but still in progress.

**Comment by geovsion** [comment]
Sorry to bother you again, how did you get your.npy file

**Comment by jiangwei221** [comment]
Which .npy do you mean?
If you are referring to the camera poses, we obtained it from the ground truth of MegaDepth dataset.
